portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function intro() {
+   document.getElementById("inner-content").style.display = "block";
+   document.getElementById("para").innerHTML = `Hi everyone! My name is Michaela Murray, and I’m currently a rising junior at Stanford University. 
+	  I am majoring in Computer Science with a concentration in Systems, and I plan to minor in mathematics 
+	  and music. So far, my favorite CS class is Operating Systems, which was taught in C. I’ve also enjoyed 
+	  my Networking class taught in C++ and a distributed systems paper-reading class where I did the final 
+	  project in Rust. Last summer, I did research in video analytics, and throughout this past year I have 
+	  continued to engage in research, the most of which was developing a sensor peripheral driver for TockOS.
+	  When I am not taking classes or doing research, my main activities are being Vice President of our school’s 
+	  security club and taking violin lessons. A fun fact about me is that I am 5’, which you might not be able to 
+	  tell over GVC.`;
+   console.log('Intro!');
+};
```

Comment:
Per [style guide](https://google.github.io/styleguide/jsguide.html#formatting-column-limit) there should be an 80 character column limit. Mind reformatting here and below?

Reply:
Noted (and updated). Thanks! Does this only apply to the javascript files?

Reply:
It's optional: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping